File: project 1 - math/js/Average.js

        let name = prompt("enter your name")
        let family = prompt("enter your family")
        let num1 = Number(prompt("num1"))
        let num2 = Number(prompt("num2"))
        let num3 = Number(prompt("num3"))
        let result = (num1 + num2 + num3) / 3
        if(result >= 17){
            alert(`${name} ${family} your course grade = A`)
        }
        else if(result >= 12 && result < 17){
            alert(`${name} ${family} your course grade = B`)
        }
        else{
            alert(`${name} ${family} your course grade = C`)
        }
  

File: project 1 - math/python/Average.py
name = input("enter your name:")
family = input("enter your family:")
num1 = float(input("enter your grade1"))
num2 = float(input("enter your grade2"))
num3 = float(input("enter your grade3"))
result = (num1 + num2 + num3) / 3
if result >=16:
    print(f"{name} {family} your graye = A")
elif result >=10 and result<=15:
    print(f"{name} {family} your graye = B")
else:
    print(f"{name} {family} your graye = C")